Streetcode/Streetcode.BLL/Mapping/AdditionalContent/TagProfile.cs

```diff
@@ -8,6 +8,6 @@ public class TagProfile : Profile
 {
     public TagProfile()
     {
-        CreateMap<Tag, TagDTO>().ReverseMap();
+        CreateMap<Tag, TagDTO>().ForMember(x => x.Streetcodes, conf => conf.Ignore());
```

Comment:
what about mapping TagDto -> Tag?
if we use it in any place , please add `CreateMap<TagDTO, Tag>();`

Reply:
For now, we use only get request, so mapping into entity is not used. Someone will add it when need it.